pages/confirmation.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
+import { NextPage } from "next";
+import { useAccount } from "@starknet-react/core";
+import { useRouter } from "next/router";
+import styles from "../styles/components/confirmation.module.css";
+import Button from "@/components/UI/button";
+import { minifyAddress } from "@/utils/stringService";
+import theme from "@/styles/theme";
+import CopyIcon from "@/components/UI/iconsComponents/icons/copyIcon";
+import DoneFilledIcon from "@/components/UI/iconsComponents/icons/doneFilledIcon";
+
+const Confirmation: NextPage = () => {
+  const router = useRouter();
+  const { address } = useAccount();
+  const tokenId: string = router.query.tokenId as string;
+  const [copied, setCopied] = useState(false);
+
+  const copyToClipboard = () => {
```

Comment:
I think we repeat this component somewhere in the app. 

Might be good to put not repeat it and make this a util in some way

Reply:
you mean the confirmation page or juste the copy address component ? 
I could only find a copy component with a different design.

Reply:
The logic of copying to clipboard something

Reply:
we set a state inside of it, I could try to send the setState inside of the function ?